### PR TITLE
Done with adding search bar.

### PR DESCRIPTION
**Pull request by SabastianMugazambi**
These are the files that I edited in the past sprint. 
- theme.liquid : comment
-home.liquid : comment
-cart.liquid  :  added feature 3

### CODE REVIEW

**Review by ammaradisa — APPROVED** [review]
Shaky but it works. Good job son.

File: HelloPullRequest.py [hunk 1]
```diff
@@ -1 +1,5 @@
-# And a function that print your name ete SabastianMugazambi({String}): print String
\ No newline at end of file
+# And a function that print your name ete SabastianMugazambi({String}): print String
+
```
Comment (CharlieAdisa) [inline HelloPullRequest.py:2]: Noice

Comment (CharlieAdisa) [inline HelloPullRequest.py:2]: derp

Comment (SabastianMugazambi) [reply HelloPullRequest.py:2]: I know I am perfect

Comment (CharlieAdisa) [reply HelloPullRequest.py:2]: another

**Review by CharlieAdisa — COMMENTED** [review]
Summary

**Review by CharlieAdisa — DISMISSED** [review]
fix

**Pull request merged**